tests/integration/cases/find_source_avaje/test.yaml

```diff
@@ -14,10 +14,20 @@ steps:
     command_args:
     - -purl
     - pkg:maven/io.avaje/avaje-prisms@1.1
+- name: Validate the produced report
+  kind: validate_schema
+  options:
+    kind: json_schema
+    schema: find_source_json_report
+    result: output/reports/maven/io_avaje/avaje-prisms/avaje-prisms.source.json
 - name: Run macaron find source with cloning enabled
   kind: find-source
   options:
     ini: config.ini
     command_args:
     - -purl
     - pkg:maven/io.avaje/avaje-prisms@1.1
+- name: Check that the repository was cloned
```

Comment:
I think I forgot another case: what happen if the step `- name: Run macaron find source` unexpectedly create a directory under `output/git_repos/github_com/avaje/avaje-prisms/` before `- name: Run macaron find source with cloning enabled` runs. 
I think to be on the safe side we can also validate that `output/git_repos/github_com/avaje/avaje-prisms/` doesn't exist after `- name: Run macaron find source` by using:
```yaml
- name: Check that the repository was NOT cloned
  kind: shell
  options:
    cmd: ls output/git_repos/github_com/avaje/avaje-prisms/
    expect_fail: true
```